### PR TITLE
[86c0dqz0q][pagination] disable input field if total number of pages is one

### PR DESCRIPTION
**Pull request by ilyabrower**
## Motivation and Context
I just disabled input if total number of pages is one.

## How has this been tested?
Manually

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [X] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [X] Nice improve.

## Checklist:

- [X] My code follows the code style of this project.
- [X] I have updated the documentation accordingly or it's not required.
- [X] Unit tests are not broken.
- [X] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

File: semcore/pagination/src/Pagination.jsx
Comment (sheila-semrush) [inline semcore/pagination/src/Pagination.jsx:1]: Doesn't work in Playground 😢 
I mean it works if TotalPages = 1 by default, but doesn't work when it's changed on the fly.
Can we do something with that?

image

Comment (ilyabrower) [reply semcore/pagination/src/Pagination.jsx:1]: fixed =)

Comment (sheila-semrush) [reply semcore/pagination/src/Pagination.jsx:1]: ahhhh lol

**Review by sheila-semrush — APPROVED** [review]
all good from my side 👍

**Review by Valeria-Zimnitskaya — APPROVED** [review]

**Pull request merged**